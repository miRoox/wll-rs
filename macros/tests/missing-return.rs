use wll_macros::{wll_export, wll_setup, wll_teardown};

#[wll_setup]
fn setup() {}

#[wll_teardown]
fn teardown() {}

#[wll_export(do_nothing)]
fn nothing() {}

fn main() {}
