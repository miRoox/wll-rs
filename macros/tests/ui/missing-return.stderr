error: invalid function signature!
  --> $DIR/missing-return.rs:10:1
   |
10 | fn nothing() {}
   | ^^^^^^^^^^^^
